src/epub/text/chapter-1.xhtml

```diff
@@ -80,7 +80,7 @@
 			<blockquote epub:type="z3998:letter">
 				<p>Blown on our beam ends. Fear position hopeless. Have already lost Maracot, Headley, Scanlan. Situation incomprehensible. Headley handkerchief end of deep sea sounding wire. God help us!</p>
 				<footer>
-					<p><abbr epub:type="z3998:initialism">S.S.</abbr> <i epub:type="se:name.vessel.ship">Stratford</i>.</p>
```

Comment:
This is not correct, S.S should not be italicized as it's a classification not the actual name of  the ship